dashboards/sqlserver/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+- category: Microsoft SQL Server
+  id: sqlserver-gce-overview
+  display_name: SQL Server GCE Overview
+  description: "This dashboard has charts for viewing MSSQL when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mssql) including User Connections, Avg Lock Wait Time, Lock Waits, Batch Info, Buffer Cache Hit Ratio, Page Checkpoint Flushes, Lazy Writes, Page Life Expectancy, Page Splits, Page Operations, CPU % Top 5 VMs, Memory % Top 5 VMs, and Hosts by Region"
+  integration_platform: GCE
```

Comment:
Can you update the format of the integration fields here? Should be `related_integrations` list. You can find documentation here https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples#metadatayaml-content

Reply:
I think should be resolved now, mind taking a second look?

Reply:
Suggested fix for `platform: GCE`, then LGTM